apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -314,7 +356,10 @@ export const EmployeeInfoPage: PageComponentType = () => {
     return (
         <EmployeePageContent
             employee={employeeWithSpecializations}
+            employeeUserId={employeeUserId}
+            organizationId={organizationId}
             updateEmployeeAction={updateEmployeeAction}
+            activeTicketsOrganizationEmployeeCount={get(activeTicketsOrganizationEmployeeCount, 'meta.count', 0)}
```

Comment:
why 0 is default here? Default means something went wrong with the query, is it not?